### PR TITLE
prep for FNO, name change and some comments

### PR DESCRIPTION
**Pull request by yairchn**
Just some name changes and added comments for Zongyi from our meeting today as preparations for OperatorFlux

### CODE REVIEW

File: src/closures/nondimensional_exchange_functions.jl [hunk 1]
```diff
@@ -73,14 +73,22 @@ function non_dimensional_function!(
...
+    # Π = [Π₁ ,Π₂ ,Π₃ ,Π₄]
+    # trafo = OF.FourierTransform(modes = (8, 8))
+    # model = Chain(
+    #     SpectralKernelOperator(trafo, 4 => 4, gelu),
+    #     SpectralKernelOperator(trafo, 4 => 2, gelu),
+    # )
+    # output = model(Π, c_gen_fno)
+    # nondim_ε = output[1,:]
+    # nondim_δ = output[2,:]
+    return nondim_ε, nondim_δ
```
Comment (charleskawczynski) [inline src/closures/nondimensional_exchange_functions.jl:89]:
```suggestion
    # Π = (Π₁ ,Π₂ ,Π₃ ,Π₄)
    # trafo = OF.FourierTransform(modes = (8, 8))
    # model = Chain(
    #     SpectralKernelOperator(trafo, 4 => 4, gelu),
    #     SpectralKernelOperator(trafo, 4 => 2, gelu),
    # )
    # model!(output, Π, c_gen_fno)
    # nondim_ε .= output[1,:]
    # nondim_δ .= output[2,:]
    return nothing
```

File: driver/generate_namelist.jl [hunk 2]
```diff
@@ -104,6 +104,12 @@ function default_namelist(case_name::String; root::String = ".", write::Bool = t
                    0.739, 0.0755, 0.178, 0.521,
                    0.0, 0.0, 0.843,-0.340,
                    0.655, 0.113, 0.0, 0.0)
+    # For FNO add here
+    # namelist_defaults["turbulence"]["EDMF_PrognosticTKE"]["general_FNO_params"] =
+    #     SA.SVector(0.3038, 0.719,-0.910,-0.483,
+    #                0.739, 0.0755, 0.178, 0.521,
+    #                0.0, 0.0, 0.843,-0.340,
+    #                0.655, 0.113, 0.0, 0.0)
```
Comment (charleskawczynski) [inline driver/generate_namelist.jl:112]: Why are we commenting this?

Comment (yairchn) [reply driver/generate_namelist.jl:112]: it's a placeholder example for Zongyi to uncomment and fill with his parameters.

Comment (charleskawczynski) [reply driver/generate_namelist.jl:112]: Ah, I just realized this is purely added comments, not commenting existing lines. Sounds good.

File: src/closures/nondimensional_exchange_functions.jl [hunk 3]
```diff
@@ -73,14 +73,22 @@ function non_dimensional_function!(
...
+
+    # Π = (Π₁ ,Π₂ ,Π₃ ,Π₄)
+    # trafo = OF.FourierTransform(modes = (8, 8))
+    # model = Chain(
+    #     SpectralKernelOperator(trafo, 4 => 4, gelu),
+    #     SpectralKernelOperator(trafo, 4 => 2, gelu),
+    # )
+    # output = model(Π, c_gen_fno)
+    # nondim_ε = output[1,:]
+    # nondim_δ = output[2,:]
```
Comment (charleskawczynski) [inline src/closures/nondimensional_exchange_functions.jl:88]:
```suggestion
    # nondim_ε .= output[1,:]
    # nondim_δ .= output[2,:]
```

**Review by charleskawczynski — APPROVED** [review]

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**